pom.xml

```diff
@@ -723,6 +723,8 @@ limitations under the License.
              change the maven.compiler.{source,target} property -->
         <configuration>
           <compilerArgument>-Xlint:deprecation</compilerArgument>
+          <source>1.8</source>
```

Comment:
isn't that already specified as properties `maven.compiler.*`

Reply:
Good question. It seems like maybe they aren't set correctly.

Note here that they should be 1.8 according to the doc [1], not 8. Let me poke around on my system more to figure out what is happening. It could be Eclipse is misinterpreting on my system or just plain user error :)

[1] https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-source-and-target.html